src/quo2/components/inputs/title_input/style.cljs

```diff
@@ -40,6 +40,7 @@
 (defn title-text
   [disabled? blur? theme]
   {:text-align-vertical :bottom
+   :padding             0
```

Comment:
hmm, probably worth giving a quick 👁️  to other uses of this component in the app

Reply:
@J-Son89 no other use cases